src/main/java/com/aws/greengrass/lambdatransformer/common/utils/LambdaComponentUtil.java

```diff
@@ -1,19 +1,28 @@
-package common.utils;
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
 
-import common.models.LambdaRuntime;
+package com.aws.greengrass.lambdatransformer.common.utils;
+
+import com.aws.greengrass.lambdatransformer.common.models.LambdaRuntime;
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
 
 import java.util.Arrays;
 import java.util.HashMap;
 import java.util.LinkedList;
 import java.util.List;
 import java.util.Map;
 
-import static common.Constants.JAVA_LAMBDA_RUNTIME_COMMAND;
-import static common.Constants.LAMBDA_EXEC_ARGS_HANDLER;
-import static common.Constants.NODEJS_LAMBDA_RUNTIME_COMMAND;
-import static common.Constants.PYTHON_LAMBDA_RUNTIME_COMMAND;
+import static com.aws.greengrass.lambdatransformer.common.Constants.JAVA_LAMBDA_RUNTIME_COMMAND;
+import static com.aws.greengrass.lambdatransformer.common.Constants.LAMBDA_EXEC_ARGS_HANDLER;
+import static com.aws.greengrass.lambdatransformer.common.Constants.NODEJS_LAMBDA_RUNTIME_COMMAND;
+import static com.aws.greengrass.lambdatransformer.common.Constants.PYTHON_LAMBDA_RUNTIME_COMMAND;
+
 
+@SuppressWarnings("PMD.DoubleBraceInitialization")
```

Comment:
Some explanation for the warning: https://pmd.github.io/latest/pmd_rules_java_bestpractices.html#doublebraceinitialization
You can initialize it in the normal way in the constructor. Then assign to it after passing through `Collections.unmodifiableMap`